Firestore/Protos/nanopb/google/protobuf/timestamp.nanopb.cc

```diff
@@ -36,6 +38,22 @@ const pb_field_t google_protobuf_Timestamp_fields[3] = {
 };
 
 
+std::string google_protobuf_Timestamp::ToString(int indent) const {
+    std::string header = PrintHeader(indent, "Timestamp", this);
+    std::string result;
+
+    result += PrintPrimitiveField("seconds: ", seconds, indent + 1, false);
+    result += PrintPrimitiveField("nanos: ", nanos, indent + 1, false);
+
+    bool is_root = indent == 0;
+    if (!result.empty() || is_root) {
```

Comment:
To concretely follow up on the points being discussed above: in the context of timestamps this is never what we want. The backend frequently distinguishes Timestamps that are set but have zero values from Timestamps that are unset. Having this logic here defeats our ability to inspect via the logs what we actually sent.